docs/tutorial/splitting-apps.md

```diff
@@ -0,0 +1,96 @@
+# Multi-File Applications
+
+When your CLI application grows, you can split it into multiple files and modules. This pattern helps maintain clean and organized code structure.
+
+This tutorial will show you how to use `add_typer` to create sub commands and organize your commands in multiple files.
+
+We will create a simple CLI with the following commands:
+
+- `version`
+- `users add NAME`
+- `users delete NAME`
+
+## CLI structure
+
+Here is the structure we'll be working with:
+
+```text
+mycli/
+├── __init__.py
+├── main.py
+├── users/
+│   ├── __init__.py
+│   ├── add.py
+│   └── delete.py
+└── version.py
+```
+
+`mycli` will be our package, and it will contain the following modules:
+
+- `main.py`: The main module that will import the `version` and `users` modules.
+- `version.py`: A module that will contain the `version` command.
+- `users/`: A package that will contain the `add` and `delete` commands.
+
+## Implementation
+
+Let's start implementing our CLI! We'll create the `version` module, the `main` module, and the `users` package.
+
+### Version Module (`version.py`)
+
+Let's start by creating the `version` module. This module will contain the `version` command.
+
+{* docs_src/splitting_apps/version.py *}
+
+In this file we are creating a new Typer app instance for the version command. This is not required in single-file applications, but in the case of multi-file applications it will allow us to include this command in the main application using `add_typer`.
+
+Let's see that next!
+
+### Main Module (`main.py`)
+
+The main module will be the entry point of the application. It will import the version module and the users module. We'll see how to implement the user module in the next section.
```

Comment:
```suggestion
The main module will be the entry point of the application. It will import the version module and the users module.

/// tip

We'll see how to implement the user module in the next section.

///
```